<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1839</width>
    <height>875</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ImageModi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>921</width>
      <height>741</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::NoDragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="AddButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RemoveButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="num_table_in">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>240</y>
      <width>871</width>
      <height>181</height>
     </rect>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>261</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>35</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>35</number>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>Size</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Spacing</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Origin</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Flip Axis</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>X</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Y</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Z</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="num_table_out">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>620</y>
      <width>871</width>
      <height>181</height>
     </rect>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>261</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>35</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>35</number>
    </attribute>
    <row>
     <property name="text">
      <string>Size</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Spacing</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Origin</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Flip Axis</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>X</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Y</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Z</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="literal_table_in">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>60</y>
      <width>871</width>
      <height>181</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>42</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>766</number>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>45</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>45</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>Path</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Data Type</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Dimension</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Input File</string>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="background">
      <color>
       <red>255</red>
       <green>170</green>
       <blue>127</blue>
      </color>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="literal_table_out">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>440</y>
      <width>871</width>
      <height>181</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>42</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>766</number>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>45</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>45</number>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>Path</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Data Type</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Dimension</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Output File</string>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select All</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ListwidgetInfoText">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1839</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionSave_Files"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_ImageModi"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_modify_images"/>
    <addaction name="actionMerge_Images"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Open&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Save&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open Images</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSave_Files">
   <property name="text">
    <string>Save Images</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action_modify_images">
   <property name="text">
    <string>Modify Images</string>
   </property>
  </action>
  <action name="actionChange_Dimension">
   <property name="text">
    <string>Change Dimension</string>
   </property>
  </action>
  <action name="actionRead_Images">
   <property name="text">
    <string>Read Images</string>
   </property>
  </action>
  <action name="actionFlip_Image_Axis">
   <property name="text">
    <string>Modify Image </string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionModify_Images">
   <property name="text">
    <string>Modify Image DataType</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionModitfy_Dimension">
   <property name="text">
    <string>Modify Image Dimension</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionModify_Image_Size">
   <property name="text">
    <string>Modify Image Size</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionModify_Image_Spacing">
   <property name="text">
    <string>Modify Image Spacing</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionModify_Image_Origin">
   <property name="text">
    <string>Modify Image Origin</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionMerge_Images">
   <property name="text">
    <string>Merge Images</string>
   </property>
  </action>
  <action name="actionSplit_Image">
   <property name="text">
    <string>Split Image</string>
   </property>
  </action>
  <action name="actionAbout_ImageModi">
   <property name="text">
    <string>About ImageModi</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
